x-pack/plugin/core/src/main/resources/monitoring-alerts-7.json

```diff
@@ -1,6 +1,6 @@
 {
   "index_patterns": [ ".monitoring-alerts-${monitoring.template.version}" ],
-  "version": 7000099,
+  "version": 7000199,
```

Comment:
Just for my understanding, what's the structure of this number? How does it correspond to the actual x.y.z version?

Reply:
The number comes from the version semantics defined by ES:

https://github.com/elastic/elasticsearch/blob/f48ddd53fd66cfef3032a17023cd9e2f88702af9/server/src/main/java/org/elasticsearch/Version.java#L42-L46

It's probably easier to read if you ignore the `99`, and it's just `70001`, with `7.00.01` representing `7.0.1` because we don't zeropad our single digit numbers.

Reply:
So, given that this PR is targetting `7.x`, shouldn't this be `7010099`?

Reply:
The current number makes sense per your other comment.